gnat2goto/driver/tree_walk.adb

```diff
@@ -21,13 +21,18 @@ with Ada.Exceptions;
 with GNAT2GOTO.Options;
 with Urealp; use Urealp;
 
+with Range_Check; use Range_Check;
+
 package body Tree_Walk is
 
    procedure Add_Entity_Substitution (E : Entity_Id; Subst : Irep);
 
    procedure Append_Declare_And_Init
      (Symbol : Irep; Value : Irep; Block : Irep; Source_Loc : Source_Ptr);
 
+   function Add_To_Table (Name : String; Symbol_Type : Irep; Value : Irep)
```

Comment:
Could use a slightly clearer name. Maybe `New_Symbol_Table_Entry` or something like that?